core/blockchain.go

```diff
@@ -149,24 +149,25 @@ const (
-	StateHistory         uint64                // Number of blocks from head whose state histories are reserved.
-	StateScheme          string                // Scheme used to store ethereum states and merkle tree nodes on top
-	PathNodeBuffer       pathdb.NodeBufferType // Type of trienodebuffer to cache trie nodes in disklayer
-	ProposeBlockInterval uint64                // Propose block to L1 block interval.
-	EnableProofKeeper    bool                  // Whether to enable proof keeper
-	KeepProofBlockSpan   uint64                // Block span of keep proof
-	SnapshotNoBuild      bool                  // Whether the background generation is allowed
-	SnapshotWait         bool                  // Wait for snapshot construction on startup. TODO(karalabe): This is a dirty hack for testing, nuke it
-	JournalFilePath      string
-	JournalFile          bool
+	TrieCleanLimit          int                   // Memory allowance (MB) to use for caching trie nodes in memory
+	TrieCleanNoPrefetch     bool                  // Whether to disable heuristic state prefetching for followup blocks
+	TrieDirtyLimit          int                   // Memory limit (MB) at which to start flushing dirty trie nodes to disk
+	TrieDirtyDisabled       bool                  // Whether to disable trie write caching and GC altogether (archive node)
+	TrieTimeLimit           time.Duration         // Time limit after which to flush the current in-memory trie to disk
+	SnapshotLimit           int                   // Memory allowance (MB) to use for caching snapshot entries in memory
+	Preimages               bool                  // Whether to store preimage of trie key to the disk
+	NoTries                 bool                  // Insecure settings. Do not have any tries in databases if enabled.
+	StateHistory            uint64                // Number of blocks from head whose state histories are reserved.
+	StateScheme             string                // Scheme used to store ethereum states and merkle tree nodes on top
+	PathNodeBuffer          pathdb.NodeBufferType // Type of trienodebuffer to cache trie nodes in disklayer
+	ProposeBlockInterval    uint64                // Propose block to L1 block interval.
+	EnableProofKeeper       bool                  // Whether to enable proof keeper
+	KeepProofBlockSpan      uint64                // Block span of keep proof
+	SnapshotNoBuild         bool                  // Whether the background generation is allowed
+	SnapshotWait            bool                  // Wait for snapshot construction on startup. TODO(karalabe): This is a dirty hack for testing, nuke it
+	JournalFilePath         string                // The file path to journal pathdb diff layers
+	JournalFile             bool                  // Whether to enable journal file
+	EnableRecoverDiffLayers bool                  // Whether enable recover diff layers for node buffer list.
```

Comment:
EnableRecoverNodebufferList->EnableRecoverDiffLayers. Here is not to recover PBSS diff layers.

Reply:
Fixed